docs/src/architectures/nanopet.rst

```diff
@@ -19,7 +19,7 @@ directory of the repository:
 
 .. code-block:: bash
 
-    pip install .[nanopet]
+    pip install metatrain[nanopet]
```

Comment:
The installation for this one remains from source for now, right?

Informally, `pip install metatrain` will be sufficient to run nanopet, because it has no extra dependencies, but updates with nanopet (flash attention, better accuracy) will come soon and won't be there in the released version

Reply:
Ah, nvm, I got confused. Let's keep it like this and then do a different release for the updates when it's time

Reply:
All good